config/config-logging.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-logging
+  namespace: triggermesh
+data:
+  _example: |
```

Comment:
How about having a real productive ConfigMap instead of this large example?
Historically people have been confused in Knative and many tried to edit the example instead of the real config, wondering why nothing was getting applied.
That's why I suggest having a real config.

If we are afraid that people are going to `kubectl apply` over their own change, let's move it to a sub-directory like `config/configmaps` or whatever.

Reply:
Ok, that makes sense.